src/clients/node/scripts/publish.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Ah, classic.

Reply:
Yeah, there were a bunch of other things here down to different npm versions too :sweat: